setup.rst

```diff
@@ -92,6 +92,9 @@ i.e., a version in :ref:`maintenance mode <maintbranch>`, you can checkout
 a release branch. For instance, to checkout a working copy of Python 3.5,
 do ``git checkout 3.5``.
 
+If you want the ability to work on multiple checkouts of python, you can use git 
+worktrees. See :ref:`committing`'s worktree section for an example.
```

Comment:
Two ways to reference the `git_worktrees` section:

```
:ref:`git_worktrees`
```
or
```
:ref: Git worktree section <git_worktrees>
```

Build the doc to see the difference between the two.

Reply:
Perfect. Thanks for the help and the review! Updated the code review and verified the build: 
<img width="1176" alt="image" src="https://user-images.githubusercontent.com/1093911/31314802-a956a91c-abbe-11e7-8f18-aafb1a407d85.png">